test/fixtures/styledMenu/index.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * This tests our tooltip implementation.
+ */
+// tslint:disable:no-console
+import {dom, DomElementArg, makeTestId, styled, TestId} from 'grainjs';
+import {cssMenuDivider, menu, menuItem, menuItemSubmenu} from '../../../lib/menu';
+
+document.addEventListener('DOMContentLoaded', () => {
+  document.body.appendChild(setupTest());
+});
+
+const testId: TestId = makeTestId('test-');
+
+const funkyMenu = styled('div', `
+  --weaseljs-font-size: 15px;
+  --weaseljs-font-family: serif;
+  --weaseljs-background-color: DarkGray;
+  --weaseljs-color: white;
+  --weaseljs-min-width: 250px;
+  --weaseljs-box-shadow: 0 0 10px rgba(0, 0, 100, 0.5);
+  --weaseljs-border: 1px solid white;
+  --weaseljs-selected-background-color: white;
+  --weaseljs-selected-text-color: black;
+`);
+
+const options = {
+  menuCssClass: funkyMenu.className,
+};
+
+function setupTest() {
+  // Create a rectangle, with a button along each edge. Each botton will have 4
+  // differently-positioned tooltps, and we'll check that hovering over each one causes 1 tooltip
+  // to flip. We'll also include a body-attached tooltip which should overhang the box.
+  return cssExample(testId('top'),
+    dom('button', 'My Menu', menu(makeMenu, options)),
```

Comment:
So to make it more like how I envision Grist, we'd have a helper like
```
function funkyMenu(createFunc: MenuCreateFunc, options?: IMenuOptions) {
  return menu(createFunc, {menuCssClass: funkyMenu.className, ...options});
}
```

Also, I suggest moving it to the same test page as the other menu. It's illustrating the same functionality. It could just be another button that opens the funkyMenu. It would reduce duplication of things like index.html (other code could be reused too), and it just seems more useful to see such variations on one page. E.g. I'd want to know that a menu is not styled globally and that nothing prevents two different-colored menus to co-exist in one page.